src/model/device/network.rs

```diff
@@ -302,10 +302,32 @@ impl From<&NetworkPortStatus> for NetworkInterfaceStatus {
             .as_ref()
             .and_then(|routers| routers.first().cloned());
 
-        let ntp_servers = if port.dhcp == DhcpType::Client {
-            port.ntp_servers.clone()
+        // collect NTP servers. Some may come from DHCP as IpAddr, others are FQDN from
+        // network configuration. Collect both types in the same list as strings
+
+        let mut ntp_servers = vec![];
+        // collect manually configured NTP servers
+        if let Some(configured_ntp_servers) = &port.configured_ntp_servers {
+            ntp_servers.extend(configured_ntp_servers.clone());
+        }
+
+        // append NTP servers provided by DHCP
+        if is_dhcp {
```

Comment:
all of the NTP servers either IP or DHCP?

Reply:
@uncleDecart what do you mean IP or DHCP? only IPs of NTP come over DHCP

Reply:
@rucoder if port.dhcp field is DhcpType::Client, can we specify NTP manually as well?

Reply:
@uncleDecart yes, but this is not currently supported :)

Reply:
Okay, should we leave TODO with comment explaining that? F.e.

TODO: We can specify NTP manually as well, but currently it's not supported, modify this to support